Cargo.toml

```diff
@@ -353,6 +354,8 @@ leveldb-sys = { git = "https://github.com/vectordotdev/leveldb-sys.git", branch
 chrono = { git = "https://github.com/vectordotdev/chrono.git", branch = "no-default-time" }
 # Adds `Status::is_reset` test, remove after the next version is released.
 h2 = { git = "https://github.com/hyperium/h2.git", rev = "f6aa3be6719270cd7b4094ee1940751b5f4ec88e" }
+# Enables OpenSSL ENGINE support during cross compilation to musl
```

Comment:
Could we add a comment here with a link to a (new) issue in Vector for upstreaming the change?

Reply:
yep, done